pom.xml

```diff
@@ -11,9 +11,13 @@
     <name>Validating String Parameter Plugin</name>
     <version>2.5-SNAPSHOT</version>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Validating+String+Parameter+Plugin</url>
-
+ 
     <properties>
         <java.level>8</java.level>
+        <junit.version>4.12</junit.version>
+        <mockito.version>2.8.9</mockito.version>
+        <assertj.version>3.13.2</assertj.version>
+        <powermock.version>1.7.4</powermock.version>
```

Comment:
I think that some of those versions are defined in the parent pom. Do we need overwrite it?

Reply:
I had to specify those versions explicitly here, I got a warning from enforcer plugin. Also, there is a specific combination of versions required. Otherwise, I was facing a couple of incompatibility issues.

Reply:
Let me check it

Reply:
As I checked only JUnit version may be sourced from the parent pom, the rest must be specified explicitly.